CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   ([#894](https://github.com/MyIntervals/emogrifier/pull/894))
 
 ### Changed
+- Upgrade to PHPUnit 7.5
+  ([#908](https://github.com/MyIntervals/emogrifier/pull/908))
```

Comment:
Forget to edit after copy and paste? ;))

Reply:
Good catch. Obviously very much so. :-)